libs/utils/src/lib/sync/components/poll/sync-poll.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'slides-poll',
+  templateUrl: './sync-poll.component.html',
+  styleUrls: ['./sync-poll.component.css']
+})
+export class SyncPollComponent implements OnInit {
```

Comment:
Unused component

Reply:
This is coming soon, I just created so someone can add shit there